docs/configuration/unit_testing_rules.md

```diff
@@ -267,3 +267,31 @@ groups:
         summary: "Instance {{ $labels.instance }} down"
         description: "{{ $labels.instance }} of job {{ $labels.job }} has been down for more than 5 minutes."
 ```
+
+### Time within tests
+
+It should be noted that in all tests, either in `alert_test_case` or
+`promql_test_case`, the output from all functions related to the current time,
+for example the `time()` and `day_of_*()` functions, will output a consistent value
+for tests.
+
+At the start of the test evaluation, `time()` returns 0 and therefore when under test
+`time()` will return a value of `0 + eval_time` value at evaluation time of `0 +
```

Comment:
Something strange happens with the suggestion here

Reply:
hmm yeah it reads weird. there was a full stop that got inserted by your earlier suggestion but the next sentence started with just `value at evaluation time` so I removed the full stop. but it feels like a jumping disk is repeating things. I'm guessing it might slipped in because the suggestion was only touching one line and not removing the rest of the sentence. I'll fix that up just now.